core/server/models/post.js

```diff
@@ -67,8 +67,6 @@ Post = ghostBookshelf.Model.extend({
     },
 
     /**
-     * We update the tags after the Post was inserted.
-     * We update the tags before the Post was updated, see `onSaving` event.
```

Comment:
This is a rebase mistake. Not sure why the comment was deleted. Will push a fix.